src/app/views/orders/order.controller.coffee

```diff
@@ -132,7 +135,7 @@
             toastr.success('mnoe_admin_panel.dashboard.subscriptions.widget.list.toastr_success', {extraData: {subscription_name: vm.order.product.name}})
           ->
             toastr.error('mnoe_admin_panel.dashboard.subscriptions.widget.list.toastr_error', {extraData: {subscription_name: vm.order.product.name}})
-        )
+        ).finally(() -> getSubscription())
```

Comment:
Where is this `getSubscription` function defined?

Reply:
@adamaziz15 My b.